test/liquid_handle_dispense_test.py

```diff
@@ -856,6 +858,170 @@ def test_tempest_bad_chip_param(self):
+        instruction = self.protocol.liquid_handle_dispense(
+            source=self.tube.well(0),
+            destination=self.flat.well(0),
+            volume="5:uL",
+            liquid=ProteinBuffer,
+        )
+
+        assert "mode_params" not in instruction.data
+
+        instruction = self.protocol.liquid_handle_dispense(
+            source=self.tube.well(0),
+            destination=self.flat.well(0),
+            volume="5:uL",
+            rows=1,
+            columns=1,
+            liquid=ProteinBuffer,
+            device="x_mantis",
+            model="high_volume",
+            diaphragm=0,
+            nozzle_size="0.1:mm",
+            tubing="LV",
+            z_drop="0.0:mm",
+            viscosity="1",
+        )
+
+        mode_params = {
+            "x_mantis": {
+                "model": "high_volume",
+                "diaphragm": 0,
+                "nozzle_size": "0.1:mm",
+                "tubing": "LV",
+                "z_drop": "0.0:mm",
+                "viscosity": "1",
+            }
+        }
+
+        liha_shape = instruction.data["shape"]
+        assert liha_shape["rows"] == 1 and liha_shape["columns"] == 1
+        assert instruction.data["mode_params"] == mode_params
+
+    def test_mantis_other_params(self):
+        """Tests mantis low_volume params, still accepted"""
+        instruction = self.protocol.liquid_handle_dispense(
+            source=self.tube.well(0),
+            destination=self.flat.well(0),
+            volume="5:uL",
+            rows=1,
+            columns=1,
+            liquid=ProteinBuffer,
+            device="x_mantis",
+            model="low_volume",
+            diaphragm=25,
+            z_drop="0.2:mm",
+        )
+
+        mode_params = {
+            "x_mantis": {
+                "model": "low_volume",
+                "diaphragm": 25,
+                "z_drop": "0.2:mm",
+            }
+        }
+
+        liha_shape = instruction.data["shape"]
+        assert liha_shape["rows"] == 1 and liha_shape["columns"] == 1
+        assert instruction.data["mode_params"] == mode_params
+
+    def test_p200_shortname_params(self):
+        """Tests that pipette-tip-p200 is still accepted"""
+        instruction = self.protocol.liquid_handle_dispense(
+            source=self.tube.well(0),
+            destination=self.flat.well(0),
+            volume="5:uL",
+            rows=1,
+            columns=1,
+            liquid=ProteinBuffer,
+            device="x_mantis",
+            tubing="pipette-tip-p200",
+        )
+
+        mode_params = {
+            "x_mantis": {
+                "tubing": "pipette-tip-p200",
+            }
+        }
+
+        liha_shape = instruction.data["shape"]
+        assert liha_shape["rows"] == 1 and liha_shape["columns"] == 1
+        assert instruction.data["mode_params"] == mode_params
+
+    def test_mantis_bad_params(self):
+        """Tests mantis bad params"""
+        # Incorrect model param and incorrect liha shape
```

Comment:
why does this one fail?

Reply:
is this just testing that `x_mantis` without rows/columns fails?

Reply:
this one fails because it doesn't send a `row`/`col`, which defaults to `tempest` shape, _but_ I specified the `device` as `mantis`

Reply:
I'll add in the comment that it's to test the incorrect liha shape for the mantis